recipes/linetools/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "linetools" %}
+{% set version = "0.3.dev2130" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30af47d42933f6e2b3494ab1a74156ae920898bbba2ade48f15b658e109e8ee0
+  patches:
+    - setup.cfg.patch
+
+build:
+  number: 0
```

I think this could be made `noarch`?
```suggestion
  number: 0
  noarch: python
```

---

I had hoped to do that, but [the setup call has a scripts argument](https://github.com/linetools/linetools/blob/master/setup.py#L129) and the documentation say [that precludes `noarch`](http://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python).

If the documentation are incorrect I'm happy to make it `noarch`.

---

Ah yes you're right, LGTM 👍